x/datadeal/keeper/deal.go

```diff
@@ -556,3 +556,43 @@ func (k Keeper) DeactivateDeal(ctx sdk.Context, msg *types.MsgDeactivateDeal) er
 
 	return nil
 }
+
+func (k Keeper) RequestDataDeliveryVote(ctx sdk.Context, msg *types.MsgRequestDataDeliveryVote) error {
+	deal, err := k.GetDeal(ctx, msg.DealId)
+	if err != nil {
+		return sdkerrors.Wrapf(types.ErrGetDeal, err.Error())
+	}
+
+	if deal.BuyerAddress != msg.RequesterAddress {
+		return sdkerrors.Wrapf(types.ErrRequestDataDeliveryVote, "only buyer can request data delivery vote")
+	}
```

Comment:
I think that there is no need to limit so that only buyers can make a request.

Reply:
Can I make requests available to non-buyers as well? Actually, I think it's okay because the data from deliveredCid is encrypted , but it's strange to allow a person who is not a buyer to make a request. wdyt?

Reply:
I think it's ok.
In general, buyer do this. 
Also, somebody can do it on behalf of the buyer who is not familiar with crypto/blockchain.
If there is no security issue, it's ok in my opinion.

Reply:
Okay, I see. I will fix it.

Reply:
I remove this validation part. Please check it :)